GeeksCoreLibrary/Modules/HealthChecks/Services/WtsHealthService.cs

```diff
@@ -12,119 +12,122 @@ namespace GeeksCoreLibrary.Modules.HealthChecks.Services;
 
 public class WtsHealthService : IHealthCheck
```

Comment:
Im not expecting to see changing to the WTShealthcheck class at all, your changes are for adding checks for a database, wtscheck itself would be unchanged i think.